src/components/DateInput.jsx

```diff
@@ -19,14 +20,17 @@ const propTypes = forbidExtraProps({
   onFocus: PropTypes.func,
   onKeyDownShiftTab: PropTypes.func,
   onKeyDownTab: PropTypes.func,
+  onKeyDownArrowDown: PropTypes.func,
+  onKeyDownQuestionMark: PropTypes.func,
 });
 
 const defaultProps = {
   placeholder: 'Select Date',
   displayValue: '',
   inputValue: '',
   screenReaderMessage: '',
-  focused: false,
+  focused: false, // handles actual DOM focus
```

Comment:
feels like these comments shouldn't need to be duplicated within the `defaultProps` definition